templates/shared/context.go

```diff
@@ -14,6 +14,7 @@ import (
 type RuleContext struct {
 	Field pgs.Field
 	Rules proto.Message
+	MessageRules proto.Message
```

Comment:
Can we avoid using `proto.Message` and actually use the type here (since it can only be one type)?

Reply:
Yup makes sense!